riff-raff/public/docs/reference/build.json.md

```diff
@@ -4,8 +4,7 @@ The build.json file
 The type ahead of the project name and build number and also the continuous deployment feature is powered by the
 `build.json` files that are created for each project.
 
-This file is normally created by the [sbt-riffraff-artifact plugin](https://github.com/guardian/sbt-riffraff-artifact)
-or [node-riffraff-artifact plugin](https://github.com/guardian/node-riffraff-artifact) plugin.
+This file is normally created by [actions-riff-raff](https://github.com/guardian/actions-riff-raff).
```

Comment:
Wonder if we should advise against manually creating this file as it makes it harder to change the shape of it? For example, we may want to add the [name of a repository's default branch](https://docs.google.com/document/d/1KkKNLCTZ_5Y1sCy0DetpDH-8aV_SwVKqEFaN3P6A3eU/edit?tab=t.0).

Reply:
I didn't go into any detail here but have advised users not to create these by hand: https://github.com/guardian/riff-raff/pull/1451/commits/df0d28f1fd953a2bffd86d75b1f3e6a374af13c0